.github/workflows/run_tests.yml

```diff
@@ -8,12 +8,26 @@ env:
   POETRY_VERSION: 1.2.2
 
 jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs: 
+      runner-label: ${{ steps.set_runner_label.outputs.runner-label }}
+    steps:
+      - name: Set runner label
+        id: set_runner_label
+        run: |
+          if [ "${{ startsWith(matrix.os, 'rdss-measlinkbot') }}" == "true" ]; then
+            echo "::set-output name=runner-label::self-hosted,windows,x64,${{ matrix.os }}"
```

Comment:
FYI, set-output is deprecated.

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Removed.